### PR TITLE
Feat/eb basics autocomplete

### PR DESCRIPTION
**Pull request by nmassardot**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
It doesn't support eb (elasticbeanstalk from AWS) command in CLI

**What is the new behavior (if this is a feature change)?**
Added the suggestion and init autocomplete with it options

**Additional info:**

- [X] Add me as codeowner of new files (you will be added as a codeowner of the files added in this PR)

### CODE REVIEW

File: src/eb.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          args: {
+            name: "keyname",
+          },
+        },
+        {
+          name: "--modules",
+          description: "List of child directories to initialize",
+        },
+        {
+          name: ["-p", "--plataform"],
```
Comment (fedeci) [inline src/eb.ts:30]:
```suggestion
          name: ["-p", "--platform"],
```

**Review by fedeci — COMMENTED** [review]
A small typo but it looks good, thanks!

File: src/eb.ts (hunk 1 above)
Comment (nmassardot) [reply src/eb.ts]: Corrected!

**Review by mschrage — APPROVED** [review]
Quick question: is the standalone elastic beanstalk CLI different from the one bundled with the aws CLI? 

We have an autogenerated completion spec foe that here: https://github.com/withfig/autocomplete/blob/master/src/aws/ebs.ts

**Comment by nmassardot** [comment]
> Quick question: is the standalone elastic beanstalk CLI different from the one bundled with the aws CLI?
> 
> We have an autogenerated completion spec for that here: https://github.com/withfig/autocomplete/blob/master/src/aws/ebs.ts

The one I added comes from [here](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb3-cmd-commands.html), they are different, am I right?

**Comment by fedeci** [comment]
@mschrage I also had the same doubt, but seems they have different subcommands. Maybe `eb` is an older deprecated version of `aws eb`?

**Comment by nmassardot** [comment]
@fedeci I don't think it is, I have found it in many guides for deploying directly to eb. Anyways, let me know, because i have been using it the last days to make deploys in my work

**Comment by fedeci** [comment]
No problem in merging this, even if it was deprecated which would mean that still a lot of users are using this!
Thanks for your contribution🎉

**Pull request merged**